CourseCreatorSkeleton

```diff
@@ -0,0 +1,39 @@
+from django.shortcuts import render, redirect, get_object_or_404
+from django.http import HttpResponseForbidden
+from django.contrib import messages
+from .models import Course, LabSection  
+from .forms import CourseForm  
+
+
+
+class CourseCreator:
+
+    #Preconditions to this: Admin user logged in. Navigated and navigated to user creation page. Or are
+        admin and selected edit profile on user profile. Or are on your own profile and selected edit
+        user.
+    @staticmethod
+    def get(request, course_id=None):
```

Comment:
The user id will be passed in through the URL using url args. I don't think there's an example on main yet but I'm using it on my branch and can show you. Also this class already exists in "views/user_form/views.py"
Also we decided to separate the creation of sections from courses. So this form doesn't need to know anything about the sections. There will be separate forms for that.

I also can't tell if this comment is properly formatted, why are you using # instead of """?

Reply:
Also this class already exists in "views/course_form/views.py"